docs/docs-components/SandpackExample.js

```diff
@@ -72,37 +57,37 @@ function SandpackContainer({
             <SandpackCodeEditor
               wrapContent
               style={{
-                height: height > MIN_HEIGHT ? height : MIN_HEIGHT,
                 flex: layout === 'column' ? 'none' : null,
-                order: 1,
```

Comment:
@AlbertCarreras fyi, we no longer need to set `order` here - we only needed it when it was conditional.